terraform.md

```diff
@@ -81,32 +78,39 @@ resource "aws_security_group_rule" "rds_outbound_to_all" {
   protocol    = "tcp"
   cidr_blocks = ["0.0.0.0/0"]
 }
-```
 
-```hcl
-# file: module/database/outputs.tf
+resource "aws_db_instance" "default" {
+  vpc_security_group_ids = [aws_security_group.rds.id]
+  # ...
+}
 
-output "security_group_id" {
+# file module/database/outputs.tf
+output "rds_security_group_id" {
```

Comment:
Calling it RDS is a bit redundant as we are on an RDS dedicated module. As from the terraform best practice we should integrate resource type in resource name (even if that's not exactly the case in here, the idea is pretty close).

Reply:
Fair enough. 

My only thought was that even though the module is called "RDS", it might end up creating several SGs for different purposes. I guess if it only exports one, we don't need to make the name more specific. Can always be refactored later if need be.